memegen/services/image.py

```diff
@@ -22,6 +23,13 @@ def find_template(self, key):
     def create_image(self, template, text, style=None):
         image = Image(template, text, style=style)
 
+        data = {
+            'path': text.path,
+            'template': template.key,
+            'style': style,
+        }
+        redis.hmset('latest', data)
```

Comment:
Here's the refactor: https://github.com/jacebrowning/memegen/pull/157

If you wouldn't mind, please give that a quick look before I merge. You'll need to rebase after that.